pkgs/development/python-modules/coinmetrics-api-client/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildPythonPackage, fetchPypi, lib, orjson, pandas, poetry-core
+, pytestCheckHook, pytest-mock, pythonOlder, python-dateutil, requests, typer
+, websocket-client }:
+
+buildPythonPackage rec {
+  pname = "coinmetrics-api-client";
+  version = "2022.8.29.6";
+  format = "pyproject";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-EPPRKOdFbLLYw0K5C4nojR8GueekoFW7xIlwKeSV1EY=";
+  };
+
+  nativeBuildInputs = [ poetry-core ];
+
+  propagatedBuildInputs = [
+    orjson python-dateutil requests typer websocket-client
+  ];
+
+  checkInputs = [
+    pandas
+    pytestCheckHook
+    pytest-mock
+  ];
+
+  pythonImportsCheck = [ "coinmetrics.api_client" ];
+
+  passthru = {
+    optional-dependencies = {
+      complete = [ pandas ];
```

Comment:
```suggestion
      pandas = [ pandas ];
```
https://github.com/coinmetrics/api-client-python/blob/2022.08.29.6/pyproject.toml#L42-L43

Reply:
Thanks. Should the manual also be updated?

<img width="1190" alt="Screen Shot 2022-08-30 at 13 12 53" src="https://user-images.githubusercontent.com/543423/187499728-f843db73-e28f-470e-819e-7e14e8e6a07b.png">

Reply:
I think the extra for dask is named complete and adds distributed.